lib/audio/client/audio-ui.js

```diff
@@ -26,27 +25,27 @@ class AudioUi
         if(!this.audioManager.categories){
             return;
         }
-        let audioSettingsTemplate = this.uiScene.cache.html.get('audio');
-        let audioCategoryTemplate = this.uiScene.cache.html.get('audio-category');
+        let audioSettingsTemplate = this.uiSceneManager.uiSceneDriver.getCacheHtml('audio');
```

Comment:
same here, instead of getCacheHtml, use a more descriptive name like loadDomContent